dlp/snippets/Inspect/inspect_string_rep.py

```diff
@@ -0,0 +1,128 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample app that uses the Data Loss Prevention API to inspect a string, a
+local file or a file on Google Cloud Storage."""
+
+
+import argparse
+
+# [START dlp_inspect_string_rep]
+from typing import List
+
+import google.cloud.dlp
+
+
+def inspect_string(
+    project: str,
+    rep_location: str,
+    content_string: str,
+    info_types: List[str],
+    include_quote: bool = True,
+) -> None:
+    """Uses the Data Loss Prevention API to analyze strings for protected data.
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        rep_location: The location of regional endpoint to use.
+        content_string: The string to inspect.
+        info_types: A list of strings representing info types to look for.
+            A full list of info type categories can be fetched from the API.
+        include_quote: Boolean for whether to display a quote of the detected
+            information in the results.
+    Returns:
+        None; the response from the API is printed to the terminal.
+    """
+
+    # Assemble the regional endpoint url using provided rep location
+    rep_endpoint = f"dlp.{rep_location}.rep.googleapis.com"
+    client_options = {"api_endpoint": rep_endpoint}
+
+    # Instantiate a client.
+    dlp = google.cloud.dlp_v2.DlpServiceClient(client_options=client_options)
+
+    # Prepare info_types by converting the list of strings into a list of
+    # dictionaries (protos are also accepted).
+    info_types = [{"name": info_type} for info_type in info_types]
+
+    # Construct the configuration dictionary. Keys which are None may
+    # optionally be omitted entirely.
+    inspect_config = {
+        "info_types": info_types,
+        "include_quote": include_quote,
+    }
+
+    # Construct the `item`.
+    item = {"value": content_string}
+
+    # Convert the project id into a full resource id.
+    parent = f"projects/{project}/locations/{rep_location}"
+
+    # Call the API.
+    response = dlp.inspect_content(
+        request={"parent": parent, "inspect_config": inspect_config, "item": item}
+    )
+
+    # Print out the results.
+    if response.result.findings:
+        for finding in response.result.findings:
+            try:
+                if finding.quote:
+                    print(f"Quote: {finding.quote}")
+            except AttributeError:
+                pass
+            print(f"Info type: {finding.info_type.name}")
+            print(f"Likelihood: {finding.likelihood}")
+    else:
+        print("No findings.")
+
+
+# [END dlp_inspect_string_rep]
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("item", help="The string to inspect.")
+    parser.add_argument(
+        "--project",
+        help="The Google Cloud project id to use as a parent resource.",
+    )
+    parser.add_argument(
+        "--rep_location",
```

Comment:
Add a few examples of what values are valid in the help message.

Also add a default value, since all flags here are optional, but leaving this empty will make the snippet fails to run.